client/lbclient/boot/replication.js

```diff
@@ -23,17 +23,32 @@ module.exports = function(client) {
   };
 
   // setup model replication
+  var since = {push: -1, pull: -1};
```

Comment:
Is there a style guide for the style you describe? I've just been following the Google one at https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml